docs/blackjack/doc.md

```diff
@@ -0,0 +1,453 @@
+
+`DoubleUpClient.blackjackPlayerMove`
+This call should be called after `DoubleUpClient.createBlackjackGame`, followed by a `DoubleUpClient.blackjackProcessPlayerMove` and then again this one.
+The game will end after the user either chooses STAND (102) or the the total sum of their cards exceed 21.
+In order to keep track of the user's hand, you an either read the events of the response, or call `DoubleUpClient.getBlackjackTable` and read its response.
+
+**Arguments**
+This call takes an argument an object with 3 mandatory fields and one optional. `coinType`, `playerAction` and `transaction` are mandatory and `coinOpt` is optional and needed only for SPLIT or DOUBLE DOWN, this is the return of a `Transaction.splitCoins` instruction in a PTB. `playerAction` is a number with valid values from 101 to 105.
+
+Argument 1:
+Type: Object
+Fields:
+    coinType: string
+    playerAction: number
+    transaction: Transaction
+    coinOpt: TransactionObjectArgument
+
+**Sample Code**
+HIT example
+```tsx
+import {DBClient} from "path/to/DoubleUpClient";
+import {Transaction} from "@mysten/sui/transactions";
+import {getSigner} from "path/to/utility_functions";
+
+/* Every transaction needs a signer in order to be executed which is
+ a keypair. If you are creating an UI then the wallet will have the method
+ wallet.signAndExecuteTransaction with the signer automatically provided. */
+ 
+ // The signer here is the user's keypair, typically the wallet has that and you
+ // don't need this.
+ const signer = getSigner();
+ const coinType = "0x2::sui::SUI";
+ 
+ const playerHIT = async () => {
+    const tx = Transaction();
+    DBClient.blackjackPlayerMove({
+        coinType: coinType,
+        playerAction: 101,
+        transaction: tx
+    });
+    const response = await DBClient.suiClient.singAndExecuteTransaction({
+        transaction: tx,
+        signer: signer,
+        options: {
+            showEffects: true,
+            showEvents: true
+        }
+    });
+    console.log(response.effects.status.status === true);
+    console.log(response.events);
+ }
+```
+DOUBLE DOWN example
+
+```tsx
+import {DBClient} from "path/to/DoubleUpClient";
+import {Transaction} from "@mysten/sui/transactions";
+import {getSigner} from "path/to/utility_functions";
+
+/* Every transaction needs a signer in order to be executed which is
+ a keypair. If you are creating an UI then the wallet will have the method
+ wallet.signAndExecuteTransaction with the signer automatically provided. */
+ 
+ // The signer here is the user's keypair, typically the wallet has that and you
+ // don't need this.
+ const signer = getSigner();
+ const coinType = "0x2::sui::SUI";
+ const betAmount = 1_000_000_000; // this should be stored when it was first requested for the DoubleUpClient.createBlackjackGame call.
+ 
+ const playerDOUBLEDOWN = async () => {
+    const tx = Transaction();
+    const coin = tx.splitCoins(tx.gas, [tx.pure.u64(betAmount)]);
```

Comment:
[coin]

Reply:
Does this work? I think that if you split two coins this won't work:
`const [coin1, coin2] = tx.splitCoins(...)`